charts/unifi/0.0.1/questions.yaml

```diff
@@ -0,0 +1,365 @@
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: port
+                      label: "Configuration"
+                      schema:
+                        type: dict
+                        attrs:
+                          - variable: port
+                            label: "container port"
+                            schema:
+                              type: int
+                              default: 3478
+                              required: false              
+                    - variable: type
+                      label: "service type"
+                      schema:
+                        type: string
+                        default: "NodePort"
+                        enum:
+                          - value: "ClusterIP"
+                            description: "ClusterIP"
+                          - value: "NodePort"
+                            description: "NodePort"
+
+  # Enable Host Networking
+  - variable: hostNetworking
+    group: "Networking"
+    label: "Enable Host Networking"
+    schema:
+      type: boolean
+      default: false
+
+## TrueCharts Specific
+
+  - variable: appVolumeMounts
+    label: "app storage"
+    group: "Storage"
+    schema:
+      type: dict
+      attrs:
+        - variable: config
+          label: "config dataset"
+          schema:
+            type: dict
+            $ref:
+              - "normalize/ixVolume"
+            attrs:
+              - variable: enabled
+                label: "Enabled"
+                schema:
+                  type: boolean
+                  default: true
+                  required: true
+                  hidden: true
+                  editable: false
+              - variable: emptyDir
+                label: "emptyDir"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+                  editable: false
+              - variable: datasetName
+                label: "Dataset Name"
+                schema:
+                  type: string
+                  default: "config"
+                  required: true
+                  editable: false
+              - variable: mountPath
+                label: "Mount Path"
+                description: "Path to mount inside the pod"
+                schema:
+                  type: path
+                  required: true
+                  default: "/unifi"
+                  editable: false
+              - variable: hostPathEnabled
+                label: "host Path Enabled"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: hostPath
+                      label: "Host Path"
+                      schema:
+                        type: hostpath
+                        required: true
+
+
+  - variable: appIngress
```

Comment:
I'm missing the ingreses for device communication and stun.

Reply:
Please provide resource with documentation, or example, for properly setting the ingress so that I can apply the necessary changes.

Reply:
There are multiple examples available in other charts...
I told you there was a reason and the reason was I didn't have the time to do it right the first time.

It's a shame you didn't start with an easy port that lacks advanced networking, but i'm currently working on things myself. So I don't have the time to figure it out completely (thats why I didn't do it in the first place). 

I THINK entrypoints should be mostly done already, but no guarantees.

Reply:
We don't have resources, with documentation yet. I'm barely working a month on this.
And networking is always on a case-by-case basis. For sure with advanced containers like network controllers.

Reply:
You can try some examples for other ingress types available with the torrent clients, but testing it really requires manual verification. And afaik device communication is HTTP, but not 100% sure.

Reply:
I do not wish to distract you from your other processes. I'm looking for guidance to assist with the process of bringing more functioning charts to the repo. Pointing out which of your other charts have valid examples is helpful.

This is the first interaction I've had with this repo. I believe fostering positive communication will go much further with enticing others to contribute to this repo.

How do I go about manual verification.

Reply:
It's not so much about distraction, but you just picked a rather hard first try...

Even for me it's mostly copy-paste, test, trail-and-error when it comes to advacned networking settings. So there is no "magic bullet" I can give you, besides walking you through it completely (which I try to do as much as possible, but I don't have a guide or something close)

I did not communicate negative nor positive, I just told you what the current state of the repo is and already gave you quite a lot of guidance.

Manual verification means actually running and testing the App, including ingress. To make sure all features work. (well in theory you can also write custom unit-tests and such, But thats overdoing it)